.github/container/nsys_jax/nsys_jax/scripts/patch_nsys.py

```diff
@@ -899,6 +899,456 @@ def consolidate_ranges(range_df):
              "start": "Start",
 '''
 
+patch_content_2025_1_1 = r'''diff --git a/nsys_recipe/lib/args.py b/nsys_recipe/lib/args.py
```

Comment:
This is just a "rebase" of the patch in https://github.com/NVIDIA/JAX-Toolbox/pull/1323; it shouldn't be needed next time.